app/controllers/contestants_controller.rb

```diff
@@ -0,0 +1,7 @@
+class ContestantsController < ApplicationController
+  def index
+    @contestants = Contestant.all
```

Comment:
This is the only thing that we would want in this controller because it is pointing to the `Contestants` index page. 
When you are faced with adding an existing project to a contestant, you will want to build a `contestant_projects_controller` with a `create` action to build a new relationship between already existing objects (project and contestant).